stylesheets/style.scss

```diff
@@ -45,6 +45,20 @@ body {
 
   .table-content table {
     width: 100%;
+
+    a {
+      margin-right: 10px;
+    }
+
+    form.destroy-action {
+      display: inline-block;
+      margin-left: 30px;
+      a {
+        font-size: 11px;
+        font-weight: 600;
+        color: red !important;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used